modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -145,6 +145,7 @@
   "imageTag": "Image Tag",
   "imageTags": "Image Tags",
   "images": "Images",
+  "importPieces": "Import {{ type }}",
```

Comment:
I would not bother localizing this. It will be added through the import module eventually.

Reply:
Ok removed